index.html

```diff
@@ -239,12 +239,40 @@ <h2 class="CONNECT2"><a href="#">LET'S CONNECT</a> </h2>
                                      
                                   </div>
                               </div>
-                    <section>
-          
-          
-                     
+                    <section>                 
             
       </div>
+      <div class="font-contact">
+        <section class="contact-form">
+
+            <div class="flex-center">
+                <div><h2 class="title-form">Contact me</h2></div>
+            <div><p class="text-contact"> if you have an application you are interested in developing a feature that you need built a project that need coding .I'd love to help with it
+                </p>
+            </div>
+        
+            <div>
+                <form class="form0"  action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] I noticed that on click of the `get in touch` button, the form input value does not submit to formspree as expected. it shows _**form not found**_ as seen in the image below 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/222110869-9ccf763c-a9a3-47b7-88f6-3e0294e424ed.png)

As explained in the project requirement, you need to visit formspree website and create a form there, then copy the `form_id` and use it here 👉🏾 ` <form class="form0"  action="https://formspree.io/f/{form_id}" method="post">`. 
Kindly fix this